tests/test_lab_dev/test_wires.py

```diff
@@ -0,0 +1,155 @@
+# Copyright 2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Tests for wires.
+"""
+
+import pytest
+
+from mrmustard.lab_dev.wires import Wires
+
+
+class TestWires:
+    r"""
+    Tests for the ``Wires`` class.
+    """
+
+    @pytest.mark.parametrize("modes", [[0, 1], [9, 4, 10]])
+    def test_init(self, modes):
+        r"""
+        Tests the init of ``Wires`` with unambiguous modes.
+        """
+        wires = Wires(modes, modes, modes)
+
+        assert wires.modes == modes
+
+        assert list(wires.out_bra.keys()) == modes
+        assert None not in list(wires.out_bra.values())
+
+        assert list(wires.in_bra.keys()) == modes
+        assert None not in list(wires.in_bra.values())
+
+        assert list(wires.out_ket.keys()) == modes
+        assert None not in list(wires.out_ket.values())
+
+        assert list(wires.in_ket.keys()) == modes
+        assert list(wires.in_ket.values()) == [None] * len(modes)
+
+    def test_init_ambiguous(self):
+        r"""
+        Tests the init of ``Wires`` with ambiguous modes.
+        """
+        wires = Wires([0, 2], [9, 3], [4, 5])
```

Comment:
I think anything should be allowed at the `Wires` level

Reply:
What do you mean?